src/SetupRHost/Product.wxs

```diff
@@ -49,6 +49,8 @@
     <DirectoryRef Id="INSTALLFOLDER">
       <Component Id="ProductAssemblies" Guid="{AFE50A8E-1F80-409E-A14C-6D940625C78F}">
         <File Source="$(var.BinDir)\Microsoft.R.Host.exe" />
+        <File Source="$(var.BinDir)\zip.dll" />
```

Comment:
I don't recall if this was in one of the past iterations or not, but can you also check that there are `<VSIXSourceItem>` entries for these DLLs inside Microsoft.VisualStudio.R.Package.csproj? If not, they should be added.